File: Day-08/Pattern4.js
function PatternFour(n) {
  let i = 1;
  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= n) {
      str += n - j + 1;
      j++;
    }
    console.log(str);
    i++;
  }
}

PatternFour(3);

File: Day-08/Pattern4.py
n = int(input())

def PatternFour(n):
    i = 1
    while(i <= n):
        j = 1
        while (j <=n):
            print(n-j+1, end='')
            j = j+1
        i= i+1
        print(end='\n')
    
PatternFour(n)